migration-engine/migration-engine-tests/tests/migrations/enums.rs

```diff
@@ -153,10 +153,34 @@ async fn variants_can_be_removed_from_an_existing_enum(api: &TestApi) -> TestRes
 
 #[test_each_connector(capabilities("enums"))]
 async fn models_with_enum_values_can_be_dropped(api: &TestApi) -> TestResult {
+    api.schema_push(BASIC_ENUM_DM).send().await?.assert_green()?;
+
+    api.assert_schema().await?.assert_tables_count(1)?;
+
+    api.insert("Cat")
+        .value("id", 1)
+        .value("mood", "HAPPY")
+        .result_raw()
+        .await?;
+
+    api.schema_push("")
+        .force(true)
+        .send()
+        .await?
+        .assert_executable()?
+        .assert_warnings(&["You are about to drop the `Cat` table, which is not empty (1 rows).".into()])?;
+
+    api.assert_schema().await?.assert_tables_count(0)?;
+
+    Ok(())
+}
```

Comment:
The diff is very confusing at the end of the file: this test was barely touched, and the last two tests are new.